lib/rules/callback-return.js

```diff
@@ -52,7 +53,7 @@ module.exports = {
          * @returns {Boolean} Whether or not this function matches our callback name.
          */
         function isCallback(node) {
-            return node.callee.type === "Identifier" && callbacks.indexOf(node.callee.name) > -1;
+            return callbacks.indexOf(sourceCode.getText(node.callee)) > -1;
```

Comment:
This doesn't actually seem to include comments (a concern raised in the original issue). See tests (tested both leading and trailing comments).